### ISSUE TITLE
Unable to upload docx file

### ISSUE THREAD

**Issue by mindaslab** [snap-dli/snap-dev-apc#1]
When I ran the app and tried to upload the provided docx file, this was the error I got

```
requests.exceptions.ConnectionError: HTTPConnectionPool(host='localhost', port=80): Max retries exceeded with url: /hsxml.lingnet.org/hsxml_Sandbox/LOG_cmd.asp (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7ff6e93bc048>: Failed to establish a new connection: [Errno 111] Connection refused',))
```

Does the code depend on external service to accomplish a task?

**Comment by e5an** [comment]
Yes. The external service is the govt's copy of the generator. By default
it links to a machine on the local network.

In the pyapp/resources folder, it's looking for a file called
custom_server_ip.txt. If that's there it will look for a copy of the
generator running at that address. If you're running it on the same
machine, that should be 127.0.0.1 or if it's another machine on the same
network, a 10.x address (for example, today my Windows laptop is  at
10.102.2.148.

**Comment by mindaslab** [comment]
Do you mean to say I must run another app along with this app? If yes can I get its source?

**Comment by e5an** [comment]
Karthik; please respond immediately if you get this message; we're having problems with email getting through.

**Comment by mindaslab** [comment]
Hello, I think I am getting all emails, including rh one you people said that its difficult to send an email.

**Comment by mindaslab** [comment]
Can we use slack or something for communication? I'm been information starved and stuck, so is the heguy handling the ASP part, he too badly needs KT.

**Comment by e5an** [comment]
Ok. I just created a Slack free account and invited you and Michael. 

We were unaware there is another guy handling the ASP part; or indeed that there was an ASP part to be handled.
